### DISCUSSION TITLE
Issues running socket.io in Docker container

### DISCUSSION THREAD

**Discussion by elmcremotebridge** [socketio/socket.io#4617] [General]
Are there any knowing issue running socket.io in a Docker container. Currently I am having a issue where there is a restart happening every 10 minutes

**Comment by qiulang** [comment]
I have been using socket.io in docker for a long time. For your problem, you need to let us know what container orchestration system you were using, e.g. is it k8s or something else? The restart seems to be caused by the orchestrator because it thinks the container is not ready, e.g. no heartbeat
